config.default.yaml

```
@@ -232,6 +232,7 @@ sector:
   marginal_cost_storage: 0. #1e-4
   methanation: true
   helmeth: true
+  coal_cc: true
```

```suggestion
  coal_cc: false
```
Let's disable this by default as the default does not feature coal power plants without carbon capture.